packages/design-system/theme.ts

```diff
@@ -159,6 +159,10 @@ const theme = {
   zIndex,
 } as const;
 
+type PalletType = typeof palette;
+type FontType = typeof fonts;
+export type PalletValueType = PalletType[keyof PalletType];
```

Comment:
pallet -> palette

Reply:
```suggestion
export type PaletteValueType = PaletteType[keyof PalletType];
```

써제션으로 리뷰 달면 편하게 반영 할 수 있어

Reply:
e915a7c00b51fc1b02e2d161f798b4a42586d57b 여기 반영ㄹ헀습니당